.eslintrc.cjs

```diff
@@ -1,5 +1,17 @@
 module.exports = {
   extends: ['@theguild'],
+  ignorePatterns: [
+    'node_modules',
+    'dist',
+    'build',
+    'generated',
+    'packages/cli/tests/cli/init',
+    'packages/cli/tests/cli/validation',
+    'packages/ts/test/',
+    'examples',
+    'cf-pages/*',
```

Comment:
`cf-pages` is no longer there, maybe others can be removed too?

Reply:
Removed :+1: